### DISCUSSION TITLE
Navidrome Home Page - The personal immersion into library

### DISCUSSION THREAD

**Discussion by mist2t** [navidrome/navidrome#3453] [Ideas]
One of the big feature currently absent in Navidrome: a default "landing" Home Page. 
Maybe it's being considered but I would love to have this.

A Home Page should be user's first interaction, pulling you in ... discovering music from the library in various ways. There are so many cool things that can be done and makes sense on a personal user homepage

Latest Added, Recently Played, Discovery functions, Various Suggestions .... and so on, sky's the limit, the list is pretty large and a bunch of cool things can be presented on the Home Page, tailored to library and user listening habits.

Just a heads up ... this might be considered already. 
I would LOVE a homepage, an immersive "cover" page into my library.

**Comment by deluan** [comment]
Yes, we know it and it will be one of the first things in the new UI. Anyway, you can have this right now by using a subsonic client like [Feishin](https://feishin.vercel.app/) or [Airsonic Refix](https://airsonic.netlify.app/). Both provide a "home" page with this kind of information.
